index.js

```diff
@@ -269,10 +269,26 @@ var PgDriver = Base.extend({
         var constraint = [],
             cb;
 
+        if ((spec.unique || spec.primaryKey) && spec.autoIncrement) {
```

Comment:
Actually this is a strange behavior. An autoIncrement should be unique, and I think the serial should too. However, releasing this constraint of an ai column being a unique or primary one, could collide with the other drivers. I don't want to have behaviors that differ from driver to driver, but I know that sometimes there are things that are specific to the db.

So could you give me a reason why one would need an ai column to be non unique?